android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -102,6 +107,14 @@ public AblyMethodCallHandler(final MethodChannel channel, final HotRestartCallba
     // paginated results
     _map.put(PlatformConstants.PlatformMethod.nextPage, this::getNextPage);
     _map.put(PlatformConstants.PlatformMethod.firstPage, this::getFirstPage);
+
+    // Encryption
```

Comment:
It's just a way to keep this file tidy, following the existing contents of the file as convention. https://github.com/ably/ably-flutter/blob/6171f2fbd23f85274b3ec865ba624c7f57b00f01/android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java#L65-L104

Reply:
Ah ok understood. that looked like a regular comment to me. For the future reference, you could specify code regions using
//region Description
//endregion

This will help IntelliJ based IDEs pick that and filter it :)